.github/workflows/goreleaser.yml

```diff
@@ -25,4 +25,4 @@ jobs:
           version: latest
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GORELEASER_GITHUB_TOKEN }}
```

Comment:
this is to allow goreleaser changes on external repository https://github.com/auth0/homebrew-auth0-cli

https://github.com/goreleaser/goreleaser/blob/68091726b31215473f0629d10e39a7e46fd0d82e/scripts/entrypoint.sh#L8-L11